compiler/rustc_error_codes/src/error_codes/E0782.md

```diff
@@ -0,0 +1,17 @@
+Trait objects must include the `dyn` keyword.
+
+Trait objects are a way to call methods on types that are not known until
+runtime but conform to some trait.
+
+In the following code the trait object should be formed with
+`Box<dyn Foo>`, but `dyn` is left off.
+
+```compile_fail,E0782
```

Comment:
```suggestion
```edition2021,compile_fail,E0782
```

Reply:
@estebank This also doesn't work as the edition is not stable yet:

```
Some expected error codes were not found: ["E0782"]
---- C:\Users\ryanl\Code\rust\build\x86_64-pc-windows-msvc\test\error-index.md - Rust_Compiler_Error_Index::E0783 (line 16023) stdout ----
error: edition 2021 is unstable and only available with -Z unstable-options.     
```

Is there a way to run with edition 2021 even when it's unstable?